interface-definitions/protocols-pim.xml.in

```diff
@@ -75,18 +171,36 @@
                   </leafNode>
                 </children>
               </tagNode>
-              <leafNode name="keep-alive-timer">
+              #include <include/pim/keep-alive-timer.xml.i>
+            </children>
+          </node>
+          <leafNode name="no-v6-secondary">
+            <properties>
+              <help>Disable IPv6 secondary address in hello packets</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <node name="spt-switchover">
+            <properties>
+              <help>Send v6 secondary addresses</help>
```

Comment:
Send shortest-path tree for IPv6
As I understand `v6 secondary address` is another option